SPECS/catch/catch.spec

```diff
@@ -35,26 +36,37 @@ is packaged up as a single header for extra convenience.
 %autosetup -n Catch2-%{version}
 
 %build
-%cmake . -Bbuild -DCATCH_ENABLE_WERROR=OFF
+%cmake . -Bbuild -DCMAKE_BUILD_TYPE=Release -DCATCH_BUILD_EXTRA_TESTS=ON -DCATCH_ENABLE_WERROR=OFF -DCATCH_INSTALL_DOCS=OFF -DBUILD_SHARED_LIBS=ON
 %make_build -Cbuild
 
 %install
 %make_install -Cbuild
-rm -rf %{buildroot}/%{_docdir}
 
 %check
 cd build
 ctest -V %{?_smp_mflags}
 
+%files            
+%license LICENSE.txt         
+%{_libdir}/libCatch2.so.%{version}   
+%{_libdir}/libCatch2Main.so.%{version}
+
 %files devel
-%license LICENSE.txt
 %doc README.md CODE_OF_CONDUCT.md docs
 %{_includedir}/catch2/
+%{_libdir}/libCatch2.so
+%{_libdir}/libCatch2Main.so
+%{_libdir}/cmake/Catch2/
 %{_datadir}/Catch2/
 %{_datadir}/pkgconfig/catch2.pc
-%{_libdir}/cmake/Catch2/
+%{_datadir}/pkgconfig/catch2-with-main.pc
 
 %changelog
+* Thu Feb 01 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 3.5.2-1
```

Comment:
Please update this line with your name and email since you made some updates to the spec file in addition to the autoupgrade tool

Reply:
Will do :)